save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Comment.kt

```diff
@@ -0,0 +1,32 @@
+package com.saveourtool.save.entities
+
+import com.saveourtool.save.spring.entity.BaseEntityWithDateAndDto
+import javax.persistence.Entity
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+/**
+ * @property message
+ * @property section
+ * @property user
+ */
+@Entity
+@Table(name = "comments")
+class Comment(
+
+    var message: String,
+
+    var section: String,
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    var user: User,
+
+) : BaseEntityWithDateAndDto<CommentDto>() {
+    override fun toDto() = CommentDto(
+        message = message,
+        userName = user.name ?: "Unknown",
```

Comment:
how we can get user.name == null?

Reply:
The _name_ field in the _User_ entity is nullable. This situation should not happen, but I think it's better to do a check than to put the operator !!